src/components/VideoPlayer/_VideoPlayer.scss

```diff
@@ -0,0 +1,31 @@
+.video-player {
+  height: 0;
+  overflow: hidden;
+  padding-bottom: 56.25%;
+  position: relative;
+
+  &--four-by-three {
+    padding-bottom: 75%;
+  }
+
+  &--square {
+    padding-bottom: 100%;
+  }
+
+  &--auto-height {
+    background-color: var(--ui-gray-warm-light);
+    height: auto;
+    padding: var(--space-s);
+  }
+
+  iframe,
+  object,
+  embed {
+    height: 100%;
+    left: 0;
+    margin: atuo;
```

Comment:
Typo. Hmmm I'm surprised `stylelint` didn't pick this up. Maybe that workflow is broken.

Reply:
Fixed.